hwp.py

```diff
@@ -86,6 +86,14 @@ def show_status(q_status: dict, t_status: dict) -> None:
             if args[1] == "submit":
                 index = args[2].rjust(3, '0')
                 file_path = args[3]
+                q_status = s.get_question_statuses()[index]
+                if q_status['release_status'] == "Closed" :
+                    print("The question has closed, "\
+                          "submitting the file anyways will delete the uploaded file "\
+                          "but won't upload your local file."\
+                          "Do you wish to continue? [y/N]", end='')
+                    if input() != "y":
+                        exit(0)
```

Comment:
probably should add `print("Aborting")` or some message

Reply:
no